adal/src/main/java/com/microsoft/aad/adal/TokenCacheItem.java

```diff
@@ -226,6 +232,23 @@ public final void setFamilyClientId(final String familyClientId) {
         this.mFamilyClientId = familyClientId;
     }
 
+    public final void setExtendedExpiresOn(final Date extendedExpiresOn) {
+        this.mExtendedExpiresOn = extendedExpiresOn;
+    }
+
+    public final Date getExtendedExpiresOn() {
+        return this.mExtendedExpiresOn;
```

Comment:
I will delete **this.**. Just wondering what should I do with the previous **this.** in this class? And is there any specific reason to avoid using **this.**? Thanks a lot. :)